### PR TITLE
fix typos in configs/

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -S *.po,*.ts,./share,./docs/man/es,./configs/attic -L ans,doubleclick,dout,halp,parms`

### CODE REVIEW

**Comment by gmoccapy** [comment]
Please be careful In merging this, as there has been made changes to glade an ui files, that might be dangerous, mostly changing ID namesNorbertVon meinem Huawei-Tablet gesendet-------- Ursprüngliche Nachricht --------Von: luzpaz <notifications@github.com>Datum: Di., 5. Jan. 2021, 20:04An: LinuxCNC/linuxcnc <linuxcnc@noreply.github.com>Cc: Subscribed <subscribed@noreply.github.com>Betreff: [LinuxCNC/linuxcnc] fix typos in configs/ (#1027)Found via codespell -q 3 -S *.po,*.ts,./share,./docs/man/es,./configs/attic -L ans,doubleclick,dout,halp,parms



You can view, comment on, or merge this pull request online at:
  https://github.com/LinuxCNC/linuxcnc/pull/1027

Commit Summary
fix typos in configs/

File Changes

    M
    configs/apps/gladevcp/README.changes
    (2)
  
    M
    configs/apps/gladevcp/colored-label/README
    (4)
  
    M
    configs/apps/gladevcp/colored-label/coloredlabel.py
    (2)
  
    M
    configs/apps/gladevcp/complex/complex.hal
    (2)
  
    M
    configs/by_interface/mesa/hm2-stepper/3x20-small.ini
    (2)
  
    M
    configs/by_interface/mesa/hm2-stepper/4i65.ini
    (2)
  
    M
    configs/by_interface/mesa/hm2-stepper/4i68.ini
    (2)
  
    M
    configs/by_interface/mesa/hm2-stepper/5i20.ini
    (2)
  
    M
    configs/by_interface/mesa/hm2-stepper/5i22-big.ini
    (2)
  
    M
    configs/by_interface/mesa/hm2-stepper/5i22-small.ini
    (2)
  
    M
    configs/by_interface/mesa/hm2-stepper/5i23.ini
    (2)
  
    M
    configs/by_interface/mesa/hm2-stepper/7i43-big.ini
    (2)
  
    M
    configs/by_interface/mesa/hm2-stepper/7i43-small.ini
    (2)
  
    M
    configs/by_interface/mesa/hm2-stepper/7i90.ini
    (2)
  
    M
    configs/by_interface/mesa/plasma-5i20/README
    (2)
  
    M
    configs/by_interface/parport/classicladder/demo_step_cl/demo_step_cl.hal
    (4)
  
    M
    configs/by_interface/parport/plasma-thc-sim/plasma_mm.ini
    (4)
  
    M
    configs/by_interface/parport/plasma-thc-sim/stepper_parport.hal
    (2)
  
    M
    configs/by_interface/parport/stepper-gantry/stepper_parport.hal
    (2)
  
    M
    configs/by_interface/parport/stepper-gantry/stepper_parport.hal.with_limits
    (2)
  
    M
    configs/by_machine/boss/boss.ini
    (2)
  
    M
    configs/by_machine/boss/pid_test.ini
    (2)
  
    M
    configs/by_machine/plasmac/plasmac_axis.py
    (2)
  
    M
    configs/by_machine/plasmac/plasmac_gcode.py
    (2)
  
    M
    configs/by_machine/plasmac/plasmac_run.py
    (2)
  
    M
    configs/by_machine/plasmac/pmx_test.py
    (2)
  
    M
    configs/by_machine/plasmac/versions.html
    (2)
  
    M
    configs/by_machine/plasmac/wizards/w_cut_recovery.py
    (6)
  
    M
    configs/by_machine/smithy/1240.hal
    (2)
  
    M
    configs/by_machine/smithy/1240_4axis.hal
    (2)
  
    M
    configs/by_machine/smithy/1240combined.hal
    (2)
  
    M
    configs/by_machine/smithy/1240combined_4axis.hal
    (2)
  
    M
    configs/by_machine/smithy/1315.hal
    (2)
  
    M
    configs/by_machine/smithy/622.hal
    (2)
  
    M
    configs/by_machine/smithy/622_4axis.hal
    (2)
  
    M
    configs/by_machine/smithy/622keyence.hal
    (2)
  
    M
    configs/by_machine/smithy/622leadshine.hal
    (2)
  
    M
    configs/by_machine/smithy/622leadshine_4axis.hal
    (2)
  
    M
    configs/by_machine/smithy/README
    (4)
  
    M
    configs/by_machine/smithy/naiky_test.hal
    (2)
  
    M
    configs/maintainer.txt
    (4)
  
    M
    configs/sim/axis/classicladder/demo_sim_cl.hal
    (2)
  
    M
    configs/sim/axis/corexy/corexy_by_hal.hal
    (2)
  
    M
    configs/sim/axis/corexy/corexy_by_kins.txt
    (2)
  
    M
    configs/sim/axis/external_offsets/circles.ngc
    (2)
  
    M
    configs/sim/axis/foam/foam.ngc
    (2)
  
    M
    configs/sim/axis/gladevcp/probe.hal
    (2)
  
    M
    configs/sim/axis/gladevcp/probe.py
    (8)
  
    M
    configs/sim/axis/ja_tests/xyzb_locking_indexer/README
    (4)
  
    M
    configs/sim/axis/ja_tests/xyzx_mpg/xhc-hb04.inc
    (4)
  
    M
    configs/sim/axis/moveoff/5_no_gui.txt
    (2)
  
    M
    configs/sim/axis/moveoff/README
    (2)
  
    M
    configs/sim/axis/moveoff/moveoff.hal
    (2)
  
    M
    configs/sim/axis/ngcgui/axis_resize.py
    (2)
  
    M
    configs/sim/axis/ngcgui/pyngcgui_axis.txt
    (2)
  
    M
    configs/sim/axis/orphans/core_sim_noio.hal
    (2)
  
    M
    configs/sim/axis/orphans/py.ini.notworking
    (2)
  
    M
    configs/sim/axis/orphans/pyiocontrol.ini.notworking
    (2)
  
    M
    configs/sim/axis/pentapod/README
    (2)
  
    M
    configs/sim/axis/remap/README
    (2)
  
    M
    configs/sim/axis/remap/extend-builtins/README
    (2)
  
    M
    configs/sim/axis/remap/getting-started/demo.ini
    (4)
  
    M
    configs/sim/axis/remap/manual-toolchange-with-tool-length-switch/nc_subroutines/manual_change.ngc
    (2)
  
    M
    configs/sim/axis/vismach/3axis-tutorial/3axis-tutorial
    (4)
  
    M
    configs/sim/axis/vismach/5axis/bridgemill/5axis.ini
    (2)
  
    M
    configs/sim/axis/vismach/VMC_toolchange/README
    (2)
  
    M
    configs/sim/axis/vismach/hexapod-sim/README
    (2)
  
    M
    configs/sim/axis/vismach/puma/README
    (2)
  
    M
    configs/sim/axis/vismach/puma/puma_cube.ini
    (4)
  
    M
    configs/sim/axis/vismach/rolfmill/rolfmill
    (4)
  
    M
    configs/sim/axis/vismach/scara/README
    (2)
  
    M
    configs/sim/gmoccapy/README
    (2)
  
    M
    configs/sim/gmoccapy/gmoccapy_plasma/README
    (2)
  
    M
    configs/sim/gmoccapy/gmoccapy_plasma/gmoccapy_plasma.ini
    (4)
  
    M
    configs/sim/gmoccapy/gmoccapy_plasma/plasma.glade
    (6)
  
    M
    configs/sim/gmoccapy/lathe_configs/lathehandler.py
    (2)
  
    M
    configs/sim/gmoccapy/macros/go_to_position.ngc
    (2)
  
    M
    configs/sim/gmoccapy/macros/i_am_lost.ngc
    (2)
  
    M
    configs/sim/gmoccapy/macros/jog_around.ngc
    (2)
  
    M
    configs/sim/gmoccapy/macros/macro_8.ngc
    (2)
  
    M
    configs/sim/gmoccapy/macros/macro_Instructions.txt
    (8)
  
    M
    configs/sim/gmoccapy/non_trivial_kinematics/scara/README
    (2)
  
    M
    configs/sim/gmoccapy/pendant.hal
    (2)
  
    M
    configs/sim/gmoccapy/plasma_config/README
    (2)
  
    M
    configs/sim/gmoccapy/plasma_config/gmoccapy_plasma.ini
    (4)
  
    M
    configs/sim/gmoccapy/plasma_config/plasma.glade
    (6)
  
    M
    configs/sim/gmoccapy/python/stdglue.py
    (4)
  
    M
    configs/sim/gscreen/README
    (2)
  
    M
    configs/sim/gscreen/gscreen_custom/gaxis_theme/gtk-2.0/gtkrc
    (4)
  
    M
    configs/sim/gscreen/gscreen_theme/gtk-2.0/gtkrc
    (2)
  
    M
    configs/sim/gscreen/silverdragon/README
    (2)
  
    M
    configs/sim/gscreen/spartan/README
    (2)
  
    M
    configs/sim/manual-example.ui
    (2)
  
    M
    configs/sim/qtvcp_screens/qtvcp_custom/custom_screencode/qtdefault_ngc/qtdefault_ngc.ui
    (6)
  
    M
    configs/sim/qtvcp_screens/qtvcp_experimental/custom_screencode/qtdefault_ngc/qtdefault_ngc.ui
    (6)
  
    M
    configs/sim/qtvcp_screens/qtvcp_experimental/lathehandler.py
    (2)
  
    M
    configs/sim/qtvcp_screens/qtvcp_experimental/qtdefault_gladevcp/qtdefault_gladevcp.ui
    (6)
  
    M
    configs/sim/touchy/gladevcp/gladevcp-touchy.hal
    (2)
  
    M
    configs/sim/touchy/touchy_postgui.hal
    (2)
  

Patch Links:
https://github.com/LinuxCNC/linuxcnc/pull/1027.patchhttps://github.com/LinuxCNC/linuxcnc/pull/1027.diff

**Comment by luzpaz** [comment]
@gmoccapy  I can revert whatever offending commits you'd like.

File: configs/sim/qtvcp_screens/qtvcp_custom/custom_screencode/qtdefault_ngc/qtdefault_ngc.ui [hunk 1]
```diff
@@ -2059,7 +2059,7 @@ border-image: url(/home/chris/emc-dev/share/qtscreen/images/frame_bg_grey.png) 0
      <property name="is_limits_overridden_status" stdset="0">
       <bool>true</bool>
      </property>
-     <property name="is_limits_overriden_status" stdset="0">
+     <property name="is_limits_overridden_status" stdset="0">
```
Comment (satiowadahc) [inline configs/sim/qtvcp_screens/qtvcp_custom/custom_screencode/qtdefault_ngc/qtdefault_ngc.ui:2062]: I could see this being potentially dangerous. Does anything else use this?

File: configs/sim/qtvcp_screens/qtvcp_experimental/custom_screencode/qtdefault_ngc/qtdefault_ngc.ui [hunk 2]
```diff
@@ -2059,7 +2059,7 @@ border-image: url(/home/chris/emc-dev/share/qtscreen/images/frame_bg_grey.png) 0
      <property name="is_limits_overridden_status" stdset="0">
       <bool>true</bool>
      </property>
-     <property name="is_limits_overriden_status" stdset="0">
+     <property name="is_limits_overridden_status" stdset="0">
```
Comment (satiowadahc) [inline configs/sim/qtvcp_screens/qtvcp_experimental/custom_screencode/qtdefault_ngc/qtdefault_ngc.ui:2062]: Likewise changing property names, does anything directly reference this?

File: configs/sim/qtvcp_screens/qtvcp_experimental/qtdefault_gladevcp/qtdefault_gladevcp.ui [hunk 3]
```diff
@@ -2059,7 +2059,7 @@ border-image: url(/home/chris/emc-dev/share/qtscreen/images/frame_bg_grey.png) 0
      <property name="is_limits_overridden_status" stdset="0">
       <bool>true</bool>
      </property>
-     <property name="is_limits_overriden_status" stdset="0">
+     <property name="is_limits_overridden_status" stdset="0">
```
Comment (satiowadahc) [inline configs/sim/qtvcp_screens/qtvcp_experimental/qtdefault_gladevcp/qtdefault_gladevcp.ui:2062]: And again

**Review by satiowadahc — COMMENTED** [review]
I think I found the only potential wrong changes to the UI, the other changes are only strings. A few of these have been bugging me as well so thanks for going through this!

**Comment by luzpaz** [comment]
bump for more exposure

**Comment by andypugh** [comment]
From inspection I am happy to merge all of these, except the changes to the .glade and .ui files. Some of these change property names. 
It might be that these _fix_ bugs if the names are spelled differently elsewhere. Or will be neutral if all instances are changed. 
It probably isn't worth changing internal code names that are invisible to the user. I support correcting spelling in labels and example files that are intended to be read by humans. 

Is it possible to split-out the changes to the .glade and .ui files for separate consideration?
Or, alternatively, confirm that you have checked the code base for all instances of the changed strings (and of the new strings, in case of collision) and see no problems?

**Comment by luzpaz** [comment]
I've removed all commits in which property tags were modified. I moved them in to a separate branch for later consideration.
https://github.com/LinuxCNC/linuxcnc/compare/master...luzpaz:API-typos

**Comment by luzpaz** [comment]
Should I squash and rebase ?

**Comment by andypugh** [comment]
There shouldn't be any need to rebase, the Github system says that it can merge automatically

**Pull request merged**